### PR TITLE
TG-23 Make a GET HTTP Request for getting a list of games

### PR DESCRIPTION
**Pull request by jpadgett7**
Just want to know if I've got the container/component architecture established correctly

### CODE REVIEW

File: containers/create_request/create_request_container.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    }, (err) => {
+      console.log(err)
+    }).catch((err) => {
+      console.log(err)
+    })
+  }
+
+  render () {
+    return (
+      <CreateRequestComponent games={this.gamesList} />
```
Comment (renodubois) [inline containers/create_request/create_request_container.js]: This should be `this.state.gamesList`

File: components/create_request/create_request_component.js [hunk 2]
```diff
@@ -36,35 +38,42 @@ export default class CreateRequestComponent extends React.Component {
         </Header>
         <Content padder>
           <Form>
+            <Item floatingLabel>
+              <Label>Post Title</Label>
+              <TextBox padder
+                     onChangeText={(input) => this.setState({postTitle: input})} />
```
Comment (renodubois) [inline components/create_request/create_request_component.js]: You can make a generic `handleInputChange` for all of your form events, the React docs go over this pretty well: https://reactjs.org/docs/forms.html#handling-multiple-inputs

**Review by renodubois — DISMISSED** [review]
Haven't ran it, but assuming it works (as much as it can w/o an actual URL).

Here are a few comments, let me know if you have questions on them.

**Review by michael-beaver — APPROVED** [review]
Tested, looks functional. We need to fix the issue with the game selection soon though.

**Pull request merged**